lib/scan/dir/engine.py

```diff
@@ -18,30 +18,26 @@
 from core._time import now
 from core.log import __log_into_file
 from core.compatible import version
+from lib.scan.dir import wordlist
+from lib.payload.wordlists import useragents
+import six
+
+
+SESSION = requests.Session()
```

Comment:
Creating session because dir_scan requests the same web multiple times. It would be good to use persistent connection.
I observed that it took some what less time in finding the directories :)